### PR TITLE
DM-44592: Remove ApTemplate from ap_pipe/ap_verify and docs

### PR DESCRIPTION
**Pull request by leeskelvin**
{Summary of changes. Prefix PR title with JIRA issue.}

****

- [x] Did you run `ap_verify.py` on this dataset?
- [x] If you made changes to `scripts/`, should those changes be propagated to [ap_verify_dataset_template](https://github.com/lsst-dm/ap_verify_dataset_template/), or are they specific to this data set?
- [x] Are the Sphinx documentation and readme up-to-date?

### CODE REVIEW

File: scripts/generate_calibs_gen3.sh
Comment (isullivan) [inline scripts/generate_calibs_gen3.sh:1]: I don't think we should remove the calibs scripts on this ticket.

File: scripts/generate_calibs_gen3_science.sh
Comment (isullivan) [inline scripts/generate_calibs_gen3_science.sh:1]: I don't think you should remove this file on this ticket.

**Review by isullivan — CHANGES_REQUESTED** [review]
Two of the calibs scripts should probably not be removed on this ticket.

**Comment by leeskelvin** [comment]
These files restored:
* `scripts/generate_calibs_gen3.sh`
* `scripts/generate_calibs_gen3_science.sh`

**Pull request merged**